src/hdf5_dl.c

```diff
@@ -313,6 +323,10 @@ herr_t H5Epush2(hid_t err_stack, const char *file, const char *func, unsigned li
     }
 }
 
+herr_t H5Eprint2( hid_t err_stack, FILE * stream ){
+    CALL(0, H5Eprint2, err_stack, stream)
```

Comment:
Use a negative value as fallback as it is what this function returns in case of error:
```suggestion
    CALL(-1, H5Eprint2, err_stack, stream)
```